system-tests/lib/resultsUtils.ts

```diff
@@ -255,7 +255,7 @@ export const expectCorrectModuleApiResult = (json, opts: {
     expectDurationWithin(
       run,
       'reporterStats.duration',
-      wallClocks,
+      wallClocks - 100,
```

Comment:
This is summing all of the attempts durations as the lower end to check the duration, but the duration of the run reported in the Module API during failures is actually ending up being 1-2 ms shorter with this update.